newrelic/resource_newrelic_alert_policy_channel.go

```diff
@@ -136,3 +166,19 @@ func policyChannelExists(client *newrelic.NewRelic, policyID int, channelID int)
 
 	return false, nil
 }
+
+func policyChannelsExist(client *newrelic.NewRelic, policyID int, channelIDs []int) (bool, error) {
```

Comment:
Perhaps here also a test would be good.

Reply:
This is covered in integration tests 😎 Did you want a unit test for this?

Reply:
Oh not if its covered.